scripts/distribution/stagenet.Dockerfile

```diff
@@ -0,0 +1,89 @@
+# syntax=docker/dockerfile:experimental
+
+# Build node.
```

Comment:
This file was deliberately deleted and the testnet one was replaced with the `builder.Dockerfile`. So this PR should not reintroduce the file.